packages/components/src/types/button-link-text.ts

```diff
@@ -0,0 +1,47 @@
+import { Generic } from '@a11y-ui/core';
+import { KoliBriButtonCallbacks, LinkTarget } from './button-link';
+import { Stringified } from './common';
+import { KoliBriIconProp } from './icon';
+import { Label } from './props/label';
+
+/**
+ * This types specifies the props of a link or button in navigations.
+ * We will support a mixin of link and button inside a navigation.
+ * Not all possible props of a link or button are relevant and supported.
+ */
+
+// do not inherit RequiredLinkProps
+type RequiredButtonProps = Label & {
+	on: KoliBriButtonCallbacks<unknown>; // actually no value is relevant
+};
+type RequiredLinkProps = Label & {
+	href: string;
+};
+type RequiredTextProps = Label;
+// do not inherit OptionalLinkProps
+type OptionalButtonOrLinkOrTextProps = {
+	active: boolean;
+	icon: Stringified<KoliBriIconProp>;
+	// tabIndex: number; // possible, but sensible ?!
+	// tooltipAlign: Alignment; // possible, but sensible ?!
+	target: LinkTarget;
+	targetDescription: string;
```

Comment:
Die beiden haben beim Button keine Wirkung, daher nicht in die allgemeinen Props.